specs/liberty/api.rst

```diff
@@ -0,0 +1,280 @@
+..
+
+This work is licensed under a Creative Commons Attribution 3.0 Unported License.
+http://creativecommons.org/licenses/by/3.0/legalcode
+
+..
+  This template should be in ReSTructured text. The filename in the git
+  repository should match the launchpad URL, for example a URL of
+  https://blueprints.launchpad.net/gslb/+spec/awesome-thing should be named
+  awesome-thing.rst .  Please do not delete any of the sections in this
+  template.  If you have nothing to say for a whole section, just write: None
+  For help with syntax, see http://sphinx-doc.org/rest.html
+  To test out your formatting, see http://www.tele3.cz/jbar/rest/rest.html
+
+=====
+ API 
+=====
+
+https://blueprints.launchpad.net/gslb/+spec/api
+
+We need an API, so defining it is a good start.
+
+Example JSON Snippets
+=====================
+
+Load Balancer JSON Snippet
+--------------------------
+
+ .. code-block:: json
+
+    {
+      "id": "f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+      "name": "Main Website",
+      "fqdn": "www.gslb.example.com.",
+      "domain_name": "example.com.",
+      "flavor": "example_1",
+      "status": "ACTIVE",
+      "links": {
```

Comment:
We might need to add the pool that is associated to the Load Balancer in the JSON snippet.